src/openrct2-ui/ride/Construction.cpp

```diff
@@ -24,6 +24,199 @@ using namespace OpenRCT2::TrackMetaData;
 
 namespace OpenRCT2
 {
+    constexpr auto kSeparator = TrackElemType::None;
+
+    /**
+     * Order of special track elements dropdown. Elements with the same name string must be sequential or they show up twice.
+     */
+    constexpr std::array kSpecialElementsDropdownOrder = {
+        TrackElemType::EndStation,
+
+        // Brakes
+        TrackElemType::Brakes,
+        TrackElemType::DiagBrakes,
+        TrackElemType::Down25Brakes,
+        TrackElemType::BlockBrakes,
+        TrackElemType::DiagBlockBrakes,
+
+        // Boosters
+        TrackElemType::Booster,
+        TrackElemType::DiagBooster,
+
+        // Photo sections
+        TrackElemType::OnRidePhoto,
+
+        // Rotation control
+        TrackElemType::RotationControlToggle,
+
+        // Brake for drop
+        TrackElemType::BrakeForDrop,
+
+        // S-Bends
+        TrackElemType::SBendLeft,
+        TrackElemType::SBendRight,
+        kSeparator,
+
+        // Tower
+        TrackElemType::TowerBase,
+        TrackElemType::TowerSection,
+
+        // Mini Golf pieces
+        TrackElemType::MinigolfHoleA,
+        TrackElemType::MinigolfHoleB,
+        TrackElemType::MinigolfHoleC,
+        TrackElemType::MinigolfHoleD,
+        TrackElemType::MinigolfHoleE,
+        kSeparator,
+
+        // (Curved) lift (hills) pieces
+        TrackElemType::LeftCurvedLiftHill,
+        TrackElemType::RightCurvedLiftHill,
+        TrackElemType::CableLiftHill,
+        TrackElemType::PoweredLift,
+        kSeparator,
```

Comment:
I don't know what happened with my earlier suggestion about moving S-Bends. In https://github.com/OpenRCT2/OpenRCT2/pull/23660/commits/1842f3724f825cf6eba84a048d5a86515f9693ec the order was Lift Hills, BrakeForDrop, then S-Bends. Somehow curved lift hills got swapped with s-bends while moving things around. This is the order I imagined:

```suggestion
        // Tower
        TrackElemType::TowerBase,
        TrackElemType::TowerSection,

        // Mini Golf pieces
        TrackElemType::MinigolfHoleA,
        TrackElemType::MinigolfHoleB,
        TrackElemType::MinigolfHoleC,
        TrackElemType::MinigolfHoleD,
        TrackElemType::MinigolfHoleE,
        kSeparator,

        // (Curved) lift (hills) pieces
        TrackElemType::LeftCurvedLiftHill,
        TrackElemType::RightCurvedLiftHill,
        TrackElemType::CableLiftHill,
        TrackElemType::PoweredLift,
        kSeparator,
        
        // S-Bends
        TrackElemType::SBendLeft,
        TrackElemType::SBendRight,
        kSeparator,
```